### ISSUE TITLE
inputdata not available

### ISSUE THREAD

**Issue by ZHONGLEI-sketch** [NOAA-GFDL/CM4#22]
run_dir and inputdata are not available from the website you provided, nor are they found on the GFDL website
![微信截图_20231219110110](https://github.com/NOAA-GFDL/CM4/assets/83412165/38a034bc-e903-48ed-8856-e62c011f7970)

**Comment by thomas-robinson** [comment]
@aradhakrishnanGFDL Do you know where the data is?  I think we need to update https://github.com/NOAA-GFDL/CM4?tab=readme-ov-file#obtaining-the-input-data

**Comment by aradhakrishnanGFDL** [comment]
@thomas-robinson Yes, please verify if this is the right one and perhaps add the links to the GitHub repo documentation directly. 

https://app.globus.org/file-manager?origin_id=5ac03700-685a-4845-ab82-4a5771b58426&origin_path=%2F1%2Fusers%2FIsaac.Held%2FCM4Documentation%2FGFDL-CM4.0%2FinputData%2F

I will open an issue for the data portal webpage fixes alongside. thanks!
